packages/ketcher-react/src/script/ui/component/form/form/form.jsx

```diff
@@ -71,18 +71,23 @@ class Form extends Component {
     onUpdate(instance, valid, errs);
   }
 
-  field(name, onChange) {
+  field(name, onChange, extraName) {
```

Comment:
Maybe rename 'extra' to something like 'second input name' or something that will describe it more clear?